README.rst

```diff
@@ -83,31 +97,11 @@ Prepare Docker
            source tempfile.sh
            rm tempfile.sh
 
-      # Verify there now is a command to 'TYPO3 render documentation full'::
+      # Verify executing the command dockrun_t3rdf::
```

Comment:
This sentence sounds a bit strange to me. I would suggest `Verify execution of the command dockrun_t3rdf works` or `Verify the command dockrun_t3rdf is available`, something like that.

The current sentence sounds like there is a word missing in the end.

Reply:
You are right, I will push another commit.